packages/core/types/src/core/controller.ts

```diff
@@ -1,7 +1,7 @@
+import type {} from 'koa-body';
```

Comment:
It doesn't work yet because empty import statements are stripped from the generated definition files.
Use the following instead to keep the import in the final build. (this does fix the issue)

```suggestion
import 'koa-body';
```